slowmovie.py

```diff
@@ -209,6 +219,7 @@ def estimate_runtime(delay, increment, frames, output="guess"):
 if not args.random_frames:
     print("Frame increment: " + str(args.increment))
```

Comment:
Any thoughts on changing everything in here to fstrings? I'm happy to do it if there's no concerns, or no reason to avoid them. I think they're easier to read (and more modern I guess).